admin_pages/events/form_sections/ConfirmEventDeletionForm.php

```diff
@@ -69,6 +75,25 @@ public function __construct($event_ids, $options_array = array())
                 'required' => true
             ]
         );
+        $events_list_url = EE_Admin_Page::add_query_args_and_nonce(
+            [
+                'status' => 'trash',
+                'return' => 'default',
+            ],
+            EVENTS_ADMIN_URL
+        );
+        $options_array['subsections']['buttons'] = new EE_Form_Section_HTML(
+            "
+        <div class='ee-admin-button-row ee-admin-button-row--align-start'>
+            <input type='submit' 
+                   class='button button--primary' 
+                   value='" . esc_attr__('Confirm', 'event_espresso') . "'
+            />
+            <a href='" . esc_url_raw($events_list_url) . ";' class='button button--secondary'>
+                " . esc_html__('Cancel', 'event_espresso') . "
+            </a>
+        </div>"
```

Comment:
```suggestion
            "<div class='ee-admin-button-row ee-admin-button-row--align-start'>
                <input type='submit' 
                       class='button button--primary' 
                       value='" . esc_attr__('Confirm', 'event_espresso') . "'
                />
                <a href='" . esc_url_raw($events_list_url) . ";' class='button button--secondary'>
                    " . esc_html__('Cancel', 'event_espresso') . "
                </a>
            </div>"
```

Reply:
i like to bump the first line down so it doesn't get appended to the previous line in the DOM
makes it easier to read when you look at the actual source code of the page (i'm old and i still do that)